### PR TITLE
Sidebar pop up on mypoke and alphabetizing props

### LINKED ISSUE #200: My Poke sidebar pop up and alphabetizing props

### PR DESCRIPTION
**Pull request by JayDampitan**
## Changes
1. Fix Sidebar popup bug on my poke. 

Closes #200

### CODE REVIEW

File: pokeapp/src/components/PokemonCard.js [hunk 1]
```diff
@@ -24,14 +24,14 @@ const PokemonCard = ({
   const [isSelected, setIsSelected] = useState(false);
 
   // ------------------------------------------------------------Pop in animation logic
-  const [active, setActive] = useState(false);
+  const [activeHover, setActiveHover] = useState(false);
```
Comment (BlackCubes) [inline pokeapp/src/components/PokemonCard.js:27]: Nice variable name change

**Review by BlackCubes — APPROVED** [review]
Let's GO!!!!!!!!!!!

**Review by Gjhnsn — APPROVED** [review]
Nice organization and detailed name changes!

**Review by Lap343 — APPROVED** [review]
Very nice looking code and what seems like you renamed does make more sense

**Pull request merged**